bgpd/bgp_packet.c

```diff
@@ -1516,7 +1517,7 @@ static int bgp_keepalive_receive(struct peer *peer, bgp_size_t size)
 
 	bgp_update_implicit_eors(peer);
 
-	peer->rtt = sockopt_tcp_rtt(peer->fd);
+	peer->rtt = sockopt_tcp_rtt(peer->connection->fd);
```

Comment:
We might use rtt inside peer_connection struct?

Reply:
as above